turingschool/savile
# CSS doc generator without plugin

**What does this PR do?**
 - Moves docs generator so that it no longer uses Jekyll plugin
 
**Where should the reviewer start?**
- `build_css_docs.rb`

**How should this be manually tested?**
- Make a new file inside of css/elements
- Write a comment that follows what you see in css/elements/button.css and some css
- run the `build_css_docs.rb` file
- look in `css_docs/elements` and you should see a new file for your new element

Any background context you want to provide?

[What are the relevant tickets?](https://3.basecamp.com/3494409/buckets/19192671/todolists/3299383724)

Screenshots (if appropriate)

Questions:
Do Migrations Need to be ran?
Do Environment Variables need to be set?
Any other deploy steps?
- Haven't addressed any of these things...
